src/generator/generator_visitor.ts

```diff
@@ -226,6 +234,10 @@ class GeneratorVisitor extends Visitor {
             this.handleShorthandAssignment(node);
         }
 
+        if (isArrayExpression(node.right)) {
```

Comment:
This should be done in the `visitArrayExpression` method.

Reply:
That's not possible, because I need the variable name to create the local array at the correct memory position.